Formula/clusterctl.rb

```diff
@@ -2,8 +2,8 @@ class Clusterctl < Formula
   desc "Home for the Cluster Management API work, a subproject of sig-cluster-lifecycle"
   homepage "https://cluster-api.sigs.k8s.io"
   url "https://github.com/kubernetes-sigs/cluster-api.git",
-      tag:      "v1.4.1",
-      revision: "39d87e91080088327c738c43f39e46a7f557d03b"
+      tag:      "v1.4.2",
+      revision: "d8c63d5edbffbd8d311fc05121bef02673159ee1"
```

Comment:
@sbueringer  Thanks fixed now. I wonder why 
`$git show-ref --tags`

generated the wrong revision though since it is what really meant for fetching the revision from the tag I thought.